src/ui/Form/Form.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Form as FormComponent } from './Form';
+
+export default {
+  title: 'Design System/Components/Form',
+  component: FormComponent,
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+    css: {
+      table: {
+        disable: true,
+      },
+    },
+    as: {
+      disable: true,
+    },
```

Comment:
```suggestion
    as: {
      table: {
        disable: true,
      },
    },
```
Please refer to the docs [here](https://storybook.js.org/docs/ember/writing-docs/doc-block-argstable#customizing) on how to disable these controls. The same needs to be done for the `ref` prop below, the point of this setting is to hide these controls as they are not needed:

<img width="1469" alt="Design_System___Components___Form_-_Form_⋅_Storybook" src="https://user-images.githubusercontent.com/78794805/199027474-ddf27a31-f66f-470d-816d-c69f545e1b35.png">

Reply:
thank you chef, always learning from you 👍

Reply:
<img width="1440" alt="Screenshot 2022-10-31 at 4 18 41 PM" src="https://user-images.githubusercontent.com/94828675/199030110-dabe7b02-41b7-4f21-b17f-514db5cbfd8b.png">